be/test/CMakeLists.txt

```diff
@@ -271,6 +267,11 @@ if ("${CMAKE_BUILD_TARGET_ARCH}" STREQUAL "x86" OR "${CMAKE_BUILD_TARGET_ARCH}"
         set_source_files_properties(./formats/json/nullable_column_test.cpp PROPERTIES COMPILE_FLAGS -mno-avx2)
 endif()
 
+if (WITH_AWS)
```

Comment:
remove WITH_AWS?

Reply:
Some UTs access AWS bucket which can not run in Aliyun Env. So I have to preseve this option to excude this UTs in Aliyun Env.